src/main/java/OutputData/editPlaylistOutputData.java

```diff
@@ -1,4 +1,22 @@
 package OutputData;
 
 public class editPlaylistOutputData {
+    String songAdded;
```

Comment:
is there a reason that these are default? if you have public getters, you could have these as private.

Reply:
The edit playlist stuff I created just to have something implemented, I updated those variable to be private now.